languages/wp-qingstor-zh_CN.po

```diff
@@ -1,11 +1,10 @@
-# Copyright (C) 2017 WP-QingStor
+# Copyright (C) 2019 WP-QingStor
 # This file is distributed under the same license as the WP-QingStor package.
 msgid ""
 msgstr ""
-"Project-Id-Version: WP-QingStor 0.3.4\n"
-"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/wp-qingstor\n"
-"POT-Creation-Date: 2017-05-17 04:23:19+00:00\n"
-"PO-Revision-Date: 2017-MO-DA HO:MI+ZONE\n"
+"Project-Id-Version: WP-QingStor 0.1.0\n"
```

Comment:
Bump this version please.

Reply:
`wp-qingstor-zh_CN.mo` should be removed?

Reply:
Modified the version already. Wp-qingstor-zh_CN.mo is needed.